packages/side-nav/test/side-nav-item.test.js

```diff
@@ -34,165 +23,144 @@ describe('side-nav-item', () => {
+        expect(item.expanded).to.be.false;
+      });
+
+      it('should expand item on first toggle button click', () => {
+        toggle.click();
+        expect(item.expanded).to.be.true;
+      });
+
+      it('should collapse item on subsequent toggle button click', () => {
+        toggle.click();
+        toggle.click();
+        expect(item.expanded).to.be.false;
+      });
+
+      it('should dispatch expanded-changed event when expanded changes', async () => {
+        const spy = sinon.spy();
+        item.addEventListener('expanded-changed', spy);
+        toggle.click();
+        await item.updateComplete;
+        expect(spy.calledOnce).to.be.true;
+      });
+    });
+
+    describe('active item with children', () => {
+      let item;
+
+      beforeEach(async () => {
+        item = fixtureSync(`
+          <vaadin-side-nav-item path="">
+            <vaadin-side-nav-item slot="children">Child 1</vaadin-side-nav-item>
+            <vaadin-side-nav-item slot="children">Child 2</vaadin-side-nav-item>
+          </vaadin-side-nav-item>
+        `);
+        await nextRender();
+        toggle = item.shadowRoot.querySelector('button');
+      });
+
+      it('should set expanded property to true by default', () => {
+        expect(item.expanded).to.be.true;
+      });
+
+      it('should collapse item on first toggle button click', () => {
+        toggle.click();
+        expect(item.expanded).to.be.false;
+      });
+
+      it('should expand item on subsequent toggle button click', () => {
+        toggle.click();
+        toggle.click();
+        expect(item.expanded).to.be.true;
+      });
     });
   });
 
-  describe('passive item', () => {
-    let passiveItem;
-
-    beforeEach(() => {
-      passiveItem = fixtureSync(`<vaadin-side-nav-item path="/another-path"></vaadin-side-nav-item>`);
-    });
+  describe('navigation', () => {
+    let anchor, toggle;
 
-    it('should not be expanded', () => {
-      expect(passiveItem.expanded).to.be.false;
+    beforeEach(async () => {
+      item = fixtureSync('<vaadin-side-nav-item></vaadin-side-nav-item>');
+      await nextRender();
+      anchor = item.shadowRoot.querySelector('a');
+      toggle = item.shadowRoot.querySelector('button');
     });
 
-    it('should not be active', () => {
-      expect(passiveItem.active).to.be.false;
+    it('should not set anchor href attribute when no path is set', () => {
+      expect(anchor.getAttribute('href')).to.be.not.ok;
     });
-  });
-
-  describe('active item', () => {
-    let activeItem;
 
-    beforeEach(() => {
-      activeItem = fixtureSync(`<vaadin-side-nav-item path=""></vaadin-side-nav-item>`);
+    it('should set empty href to the anchor when path is empty string', async () => {
+      item.path = '';
+      await nextRender();
+      expect(anchor.getAttribute('href')).to.be.empty;
     });
 
-    it('should be expanded', () => {
-      expect(activeItem.expanded).to.be.true;
```

Comment:
This test is a duplicate: we check for `expanded` to be set for the active item with and without children.
To me it feels unnecessary to repeat this assertion twice, so I removed this test.